solana-sample/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
Not really a fan of local toolchains for each project. Would be best to have these settings configured globally for all projects by default if possible